### PR TITLE
User story 002 angular

### PR DESCRIPTION
**Pull request by Christopher154**
Routing added
Listcomponent added
Proxy settings added

### CODE REVIEW

**Review by matpmag — DISMISSED** [review]
⚱️

File: Kainos-map/src/app/components/list/list.component.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  constructor(dataService: DataService) { 
+    this.data = dataService
+  }
+
+  ngOnInit() {
+    this.retrieveCourses()
+  }
+
+  retrieveCourses() {
+    this.data.getCities().subscribe(data => {
```
Comment (ConorLeckey) [inline Kainos-map/src/app/components/list/list.component.ts]: CITIES

**Review by ConorLeckey — APPROVED** [review]
:100:

**Review by matpmag — APPROVED** [review]
🍹

**Pull request merged**